lib/modules/versioning/pep440/range.ts

```diff
@@ -85,7 +85,7 @@ function getFutureVersion(
 
 interface Range {
   operator: string;
-  prefix: string;
+  prefix?: string;
```

Comment:
why this change?

Reply:
This overall change includes converting ranges with the operator `~=` down into a pair of new range objects, and I didn't feel like it was necessary to include prefixes in either of those objects (see lines 377-381).

Reply:
This [already present line](https://github.com/renovatebot/renovate/pull/15704/files#diff-f6e993a4715dd7526818c5bb2cdfdaa3a139db2aa290ba8857b9d93f8fe03c4fL290) suggests that the field should be nullable as well.

Reply:
ok